src/test/java/pages/HoversPage.yml

```diff
@@ -0,0 +1,7 @@
+urls:
+   default: http://the-internet.herokuapp.com/hovers  
+elements: 
+  user_div:
+   xpath: "(//img[@alt='User Avatar'])[1]"  
+  tooltip:
```

Comment:
The tooltip xpath should no longer be in here.

Reply:
This needs to be deleted. You cannot have a declared xpath to the tooltip or this is going to take too much work to use.